packages/wrangler/src/__tests__/pages/project-upload.test.ts

```diff
@@ -479,22 +480,3 @@ describe("pages project upload", () => {
 		expect(std.err).toMatchInlineSnapshot(`""`);
 	});
 });
-
-/**
- * When uploading assets we print out progress messages that look like:
- *
- * Uploading... (1/4)
- * Uploading... (2/4)
- * Uploading... (4/4)
- *
- * Note that we don't always get a progress message for every item uploaded.
- * These upload counts are not deterministic and can change from test run to test run.
- *
- * Also if you run the tests in --runInBand mode then we never see any of
- * these progress messages in the tests!!
- *
- * So this helper removes these message from the snapshots to keep them consistent.
- */
-export function normalizeProgressSteps(str: string): string {
```

Comment:
exporting this function and reusing it elsewhere was causing this suite to be run twice, so I extracted it into its own helper.